hawkular-wildfly-agent/src/main/java/org/hawkular/agent/monitor/extension/MonitorServiceRestartParentAttributeHandler.java

```diff
@@ -54,11 +53,6 @@ protected ServiceName getParentServiceName(PathAddress parentAddress) {
     }
 
     @Override
-    protected boolean isResourceServiceRestartAllowed(OperationContext context, ServiceController<?> service) {
```

Comment:
Maybe we could try (in a separate PR) to somehow mark the agent as `needs-restart` to let know the user that we are dealing with an agent running using old configurations.